azkaban-common/src/test/java/azkaban/jobExecutor/ProcessJobTest.java

```diff
@@ -46,6 +46,12 @@ public void setUp() throws IOException {
     props.put(AbstractProcessJob.WORKING_DIR, workingDir.getCanonicalPath());
     props.put("type", "command");
     props.put("fullPath", ".");
+    
+    props.put(CommonJobProperties.PROJECT_NAME, "test_project");
+    props.put(CommonJobProperties.FLOW_ID, "test_flow");
+    props.put(CommonJobProperties.JOB_ID, "test_job");
+    props.put(CommonJobProperties.EXEC_ID, "123");
+    props.put(CommonJobProperties.SUBMIT_USER, "test_user");
```

Comment:
Add a test 
* Remove SUBMIT_USER and add USER_TO_PROXY
* Make sure job run successfully

Add a test w/o both submit_user and user_to_proxy